.gitignore

```diff
@@ -17,6 +17,10 @@ footloose.yaml
 # OS-specific files:
 .DS_Store
 
+# IDE-specific configuration:
+.idea
+.vscode
+
```

Comment:
Suggest these belong in your personal .gitignore.

Reply:
Updated